src/v2/Components/NavBar/menuData.ts

```diff
@@ -25,20 +25,16 @@ export const ARTISTS_SUBMENU_DATA: MenuLinkData = {
     title: "Artists",
     links: [
       {
-        text: "Portrayals of Queer Love",
-        href: "/collection/queer-love-pride-month-2020",
+        text: "In-Demand Artists",
+        href: "/collection/in-demand-artists",
       },
       {
-        text: "Contemporary Women Surrealists",
-        href: "/collection/contemporary-women-surrealists",
+        text: "Artist's We're Eyeing",
```

Comment:
Ah, this should actually be "Artists We're Eyeing" (it's incorrect in the spreadsheet too, but you can see on the [actual page](https://www.artsy.net/collection/artists-we-are-eyeing) how it should read)